packages/amplify_api_rest/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Don't forget to add license.

Reply:
will update in our merged package (RestAPI + GraphQL)